.pre-commit-config.yaml

```
@@ -41,3 +41,12 @@ repos:
         language: system
         entry: python tools/generate_requirements.py
         files: "pyproject.toml|requirements/.*\\.txt|tools/generate_requirements.py"
+
+ci:
+  # This ensures that pr's aren't autofixed by the bot, rather you call
```

```suggestion
  # This ensures that PR's aren't autofixed by the bot, rather you call
```
(just more readable)